results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/WikiClusteringFastP2P.json

```diff
@@ -93,7 +93,7 @@
         ]
       }
     },
-    "evaluation_time": 43.14,
```

Comment:
Hmm didn't make much of a difference for this dataset https://github.com/embeddings-benchmark/mteb/pull/745/commits/db8ef57bc8abf314001d120e3ca8393716a7b185 

I don't really have a preference, so can keep your suggestion.